tests/acceptance/project-test.js

```diff
@@ -12,10 +12,11 @@ module("Acceptance | projects", function(hooks) {
 
   hooks.beforeEach(async function() {
     const user = this.server.create("user", { isReviewer: true });
+    this.server.create("project-assignee", {
+      user
+    });
     // eslint-disable-next-line camelcase
     await authenticateSession({ user_id: user.id });
-
-    this.server.create("project", { reviewers: [user] });
```

Comment:
Insert this again. This is the reason for the failing acceptance test of projects. If we do not have any projects, there won't be any to select from.

Reply:
A project is created via the project-assignee factory, am I right?

Reply:
Exactly